dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/runner/TaskExecuteThread.java

```diff
@@ -273,13 +275,20 @@ public void kill() {
     public void downloadResource(String execLocalPath, Logger logger, List<Pair<String, String>> fileDownloads) {
         for (Pair<String, String> fileDownload : fileDownloads) {
             try {
+                WorkerServerMetrics.incWorkerResourceDownloadCount();
                 // query the tenant code of the resource according to the name of the resource
                 String fullName = fileDownload.getLeft();
                 String tenantCode = fileDownload.getRight();
                 String resHdfsPath = storageOperate.getResourceFileName(tenantCode, fullName);
                 logger.info("get resource file from hdfs :{}", resHdfsPath);
+                Long resourceDownloadStartTime = System.currentTimeMillis();
```

Comment:
There is no need to do an extra pack
```suggestion
                long resourceDownloadStartTime = System.currentTimeMillis();
```

Reply:
Sure, will fix it. Thx

Reply:
Fixed by the latest commit.